common/views/components/ResetActiveFilters/ResetActiveFilters.tsx

```diff
@@ -0,0 +1,294 @@
+import React, { FunctionComponent, ReactElement, ReactNode } from 'react';
+import { LinkProps } from '@weco/common/model/link-props';
+import {
+  CatalogueAggregationBucket,
+  CatalogueAggregations,
+} from '@weco/common/model/catalogue';
+import {
+  worksLink,
+  imagesLink,
+  WorksRouteProps,
+} from '../../../services/catalogue/ts_routes';
+import Icon from '../Icon/Icon';
+import Space from '../styled/Space';
+import NextLink from 'next/link';
+import { font, classNames } from '../../../utils/classnames';
+import styled from 'styled-components';
+
+type ResetActiveFilters = {
+  workTypeFilters: CatalogueAggregationBucket[];
+  productionDatesFrom: string | null;
+  productionDatesTo: string | null;
+  worksRouteProps: WorksRouteProps;
+  imagesColor: string | null;
+  workTypeInUrlArray: string[];
+  aggregations?: CatalogueAggregations;
+  resetFilters: LinkProps;
+  enableMoreFilters: boolean;
+  languagesInUrl: string[];
+  subjectsInUrl: string;
+  genresInUrl: string;
+};
+
+const ColorSwatch = styled.span`
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  background-color: ${({ color }) => color};
+  margin-left: 6px;
+  padding-top: 2px;
+`;
+
+type CancelFilterProps = {
+  text?: string;
+  children?: ReactNode;
+};
+
+const CancelFilter: FunctionComponent<CancelFilterProps> = ({
+  text,
+  children,
+}: CancelFilterProps): ReactElement<CancelFilterProps> => {
+  return (
+    <Space
+      as="span"
+      h={{
+        size: 'm',
+        properties: ['margin-right'],
+      }}
+    >
+      <Space
+        as="span"
+        h={{
+          size: 'xs',
+          properties: ['margin-right'],
+        }}
+      >
+        <Icon
+          name="cross"
+          extraClasses="icon--match-text icon--silver v-align-middle"
+        />
+      </Space>
+      <span className="visually-hidden">remove </span>
+      {text || children}
+    </Space>
+  );
+};
+
+export const ResetActiveFilters: FunctionComponent<ResetActiveFilters> = ({
+  workTypeFilters,
+  productionDatesFrom,
+  productionDatesTo,
+  worksRouteProps,
+  imagesColor,
+  workTypeInUrlArray,
```

Comment:
This name is very odd? not just `workType`?

Reply:
Yeah this was existing naming before. I will update this as mentioned in our call to see if we can leverage on workprops rather than passing it down as if we want a new filter we would have todo the same.